src/tools/circle/circletool.cpp

```diff
@@ -36,36 +36,19 @@ QString CircleTool::description() const
 
 CaptureTool* CircleTool::copy(QObject* parent)
 {
-    return new CircleTool(parent);
+    auto* tool = new CircleTool(parent);
```

Comment:
Won't this cause a slow memory leak? I don't see where either the original or new copy is destructed.

Reply:
I'll check it tomorrow or the day after tomorrow. I bit tired after the long road today.

Reply:
In general nothing changed here, 
previous code:

```
CaptureTool* CircleTool::copy(QObject* parent)
{
    return new CirceTool(parent);
}
```

Now we have:
```
CaptureTool* CircleTool::copy(QObject* parent)
{
    auto* tool = new CircleTool(parent);
    copyParams(this, tool);
    return tool;
}
```

The code is still returning the same pointer.
I even did a check to be sure. I have placed a huge 1Gb array into the CircleTool and each new object takes it and I can see it, after closing CaptureWidget memory utilisation by Flameshot backs to 24.3Mb.